packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DrawingTask/icons.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react'
+import { Blank } from 'grommet-icons'
+
+const BlankWithViewBox = ({ children }) => (
+  <Blank viewBox='0 0 100 100'>
+    {children}
+  </Blank>
+)
+
+const icons = {
```

Comment:
maybe something like [getViewer](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getViewer/getViewer.js) would be preferable? Therefore split icons into individual files.

Reply:
I think so. At some point I think we're going to move the task types into their own package, so the consuming app only loads up what it needs.